increase_min_pass_length.diff

```diff
@@ -0,0 +1,39 @@
+diff --git a/custom/conf/app.example.ini b/custom/conf/app.example.ini
```

Comment:
why is this file here? 🤔

Reply:
Thanks for catching this. I copied the patch from another machine to create the commit and push it, and forgot to rm this file.